python/apps/taiga/src/taiga/workflows/repositories.py

```diff
@@ -165,51 +166,99 @@ def _apply_filters_to_workflow_status_queryset(
     return qs.filter(**filter_data)
 
 
+WorkflowStatusSelectRelated = list[
+    Literal[
+        "workflow",
+    ]
+]
+
+
+def _apply_select_related_to_workflow_status_queryset(
+    qs: QuerySet[WorkflowStatus],
+    select_related: WorkflowStatusSelectRelated,
+) -> QuerySet[WorkflowStatus]:
+    return qs.select_related(*select_related)
+
+
+WorkflowStatusOrderBy = list[
+    Literal[
+        "order",
+        "-order",
+    ]
+]
+
+
+def _apply_order_by_to_workflow_status_queryset(
+    qs: QuerySet[WorkflowStatus], order_by: WorkflowStatusOrderBy
+) -> QuerySet[WorkflowStatus]:
+    return qs.order_by(*order_by)
+
+
 ##########################################################
 # WorkflowStatus - create workflow status
 ##########################################################
 
 
 def create_workflow_status_sync(
     name: str,
-    slug: str,
+    slug: str | None,
     color: int,
     order: int,
     workflow: Workflow,
 ) -> WorkflowStatus:
-
-    return WorkflowStatus.objects.create(
+    status = WorkflowStatus(
         name=name,
-        slug=slug,
         color=color,
         order=order,
         workflow=workflow,
     )
+    if slug:
+        status.slug = slug
+
+    status.save()
+
+    return status
 
 
 create_workflow_status = sync_to_async(create_workflow_status_sync)
```

Comment:
is it still necessary?

Reply:
yes. Because we create statuses with an inital template